patterns/page-creation-pattern.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * Title: A page creation pattern.
+ * Slug: elementary-theme/page-creation-pattern
+ * Block Types: core/post-content
+ *
+ * @package Elementary-Theme
+ */
```

Comment:
The WP native function `register_block_pattern` is used to accomplish this prior to WP 6.0, and now it is possible to register a block pattern from the `patterns` directory itself.

https://github.com/rtCamp/theme-elementary/blob/ba0bef456a00f8accf342702c6fec8f02f173563/inc/classes/patterns/class-block-patterns.php#L107-L110

Either we relocate our other block patterns to the `patterns` directory or we keep this theme compatible with WP 5.9 and register block patterns with `register_block_pattern`.

Additionally, rather than a page titled `A page creation pattern` if we are going to include page creation block patterns, they should be general ones that are utilized frequently, such as `Contact Page` and `About Page`.